packages/graphile-build-pg/src/plugins/PgColumnDeprecationPlugin.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+import type { Plugin } from "graphile-build";
+
+export default (function PgColumnDeprecationPlugin(builder) {
+  builder.hook("GraphQLObjectType:fields:field", (field, build, context) => {
+    const {
+      scope: {
+        isPgRowType,
+        isPgCompoundType,
+        pgIntrospection: table,
+        pgFieldIntrospection,
+      },
+    } = context;
+    if (
+      !(isPgRowType || isPgCompoundType) ||
+      !table ||
+      table.kind !== "class" ||
+      !pgFieldIntrospection ||
+      typeof pgFieldIntrospection.tags !== "object"
```

Comment:
I'll edit this in a minute because `typeof null === 'object'` and then line 23 would crash. I'll just replace with a truthy check on `pgFieldIntrospection.tags`